api/server/router/local/image.go

```diff
@@ -433,6 +425,39 @@ func (s *router) postBuild(ctx context.Context, w http.ResponseWriter, r *http.R
 	return nil
 }
 
+// repoAndTag is a helper struct for holding the parsed repositories and tags of
+// the input "t" argument.
+type repoAndTag struct {
+	repo, tag string
+}
+
+// parseRepoAndTags parses the raw "t" argument received from the client
+// to a slice of repoAndTag.
+// It also validates each repoName and tag.
+func parseRepoAndTags(names []string) ([]repoAndTag, error) {
```

Comment:
Would like to see some sanitization in here. What would the expected behaviour be if I provided `-t endophage/alpine -t endophage/alpine:latest`? I would be happy with raising an error if the user provides ambiguous/colliding options.